docs/guides/debugging/clion.md

```diff
@@ -0,0 +1,51 @@
+
+## Top-level Cargo Workspace
+
+By default Tauri places the Rust project in a subdirectory called `src-tauri`. CLion doesn't recognize Cargo projects that are not at the top-level like this, so we need to create a top-level Workspace that just points to our main `Cargo.toml` file: 
+
+```toml title=Cargo.toml
+[workspace]
+members = ["src-tauri"]
+```
+
+Before you proceeed, make sure that your project is fully loaded. If the indexing is finished and the Cargo tool window shows all the modules and targets of the workspace, you’re good to go.
+
+## Run/Debug Configuration
+
+We will set up a Run/Debug configuration that we can use to launch our Tauri app in debugging mode. To create a configuration, go to Edit Configurations, click **+**, and then select Cargo Command.
+
+![Add Run/Debug Configuration](/img/guides/debugging/clion/add-cargo-config-light.png#gh-light-mode-only)
+![Add Run/Debug Configuration](/img/guides/debugging/clion/add-cargo-config-dark.png#gh-dark-mode-only)
+
+With that created we need to configure CLion so it instructs Cargo to build our app without any default features. This will tell Tauri to use your development server instead of reading assets from disk. Normally this flag is passed by the Tauri CLI, but since we're completely sidestepping that here, we need to pass the flag manually.
+
+![Add `--no-default-features` flag](/img/guides/debugging/clion/set-no-default-features-light.png#gh-light-mode-only)
+![Add `--no-default-features` flag](/img/guides/debugging/clion/set-no-default-features-dark.png#gh-dark-mode-only)
+
+Now we can optionally rename the Run/Debug Configguration to something more memorable, in this example we called it "Run Tauri App", but you can name it whatever you want.
+
+![Rename Configuration](/img/guides/debugging/clion/rename-configuration-light.png#gh-light-mode-only)
+![Rename Configuration](/img/guides/debugging/clion/rename-configuration-dark.png#gh-dark-mode-only)
+
+## Launch the Development Server
+
+The above configuration will use Cargo directly to build the Rust application and attach the debugger to it. This means we completely sidestep the Tauri CLI, so features like the `beforeDevCommand` and `beforeBuildCommand` will **not** be executed. We need to take care of that by opening a new Terminal and starting running the development server manually:
+
+```sh
+pnpm vite dev
+```
+
+> Note that CLion currently has no support for background tasks in Run/Debug configurations like VS Code has, so for now you need to manually run the development server.
+
+## Launch a Debugging Session
+
+With the development server running and the Run/Debug Configuration selected in the Switcher, you can now start a new debugging session by clicking Debug `Ctrl+D` on Windows/Linux, `⌃D` on macOS). CLion will automatically recognize breakpoints placed in any Rust file in your project.
```

Comment:
The shortcuts seem to be wrong, for me it lists these:
![grafik](https://user-images.githubusercontent.com/30730186/210239316-4bef213b-f578-4acf-86f1-81c5089424ac.png)

Reply:
I took these from their official docs 🤔 maybe it's a weird Germany specific keymap? When you go to settings > keymap what does it say your keymap is?

( for me it says "macOS" for example, but CLion has a number of presets that can emulate an editor basically)

Reply:
Keymap: `Windows` (there's no macos or linux option so i can't check if it's Windows specific)

Reply:
Yeah I think this is a windows specific setting then, maybe their docs are outdated

Reply:
Maybe it's better to focus on the Button (for the mouse) for the guide then? Since it shows the shortcut people can connect the dots themselves.

Reply:
Yeah let's just remove the shortcuts
```suggestion
With the development server running and the Run/Debug Configuration selected in the Switcher, you can now start a new debugging session by clicking Debug. CLion will automatically recognize breakpoints placed in any Rust file in your project.
```